templates/default/onlinefs-site.xml.erb

```diff
@@ -34,6 +38,12 @@
     <url><%= @hopsworks_url %></url>
   </hopsworks>
 
+  <opensearch>
+    <host><%= node['onlinefs']['opensearch']['host'] %></host>
+    <userName><%= node['onlinefs']['opensearch']['user_name'] %></userName>
+    <password><%= node['onlinefs']['opensearch']['password'] %></password>
+  </opensearch>
```

Comment:
I think here we should use certificates instead of passwords. Users have asked us to remove passwords from config files.

Reply:
Can you point me to some reference e.g PR of using certificate for authentication? From what I see, other services are using password as well: https://github.com/logicalclocks/elasticsearch-chef/blob/master/attributes/default.rb#L103-L116